src/compiler/compile/css/Stylesheet.ts

```diff
@@ -1,3 +1,17 @@
+// TODO move pushArray to npm module
```

Comment:
Svelte tries to minimize dependencies, so I don't know if this should be in an npm module, but it'd be nice to make it in a file imported and shared everywhere it's use